problog/formula.py

```diff
@@ -1411,11 +1411,12 @@ def get_body(self, index, processed=None, parent_name=None):
                 else:
                     return node.name
             elif ntype == "conj":
+                children = self._unroll_conj(node)
+                result = And.from_list(list(map(self.get_name, children)))
                 if index < 0:
-                    return -node.name
+                    return -result
```

Comment:
Not sure on this.

Reply:
node.name was throwing an exception when doing `\+ ( \+heads1, \+ heads2)` in `some_heads.pl` and converting it to a Bayesian Network, due to the node not having a name. I must admit that I was also not entirely sure of this solution, but it's what @rmanhaeve suggested me to do when we had this error.

Reply:
It seems reasonable to expect the body as the result...
What about the ntype=='disj' case? Shouldn't we change that as well?

Reply:
Does "result" contain the body, or is it in another variable?
The "disj" case indeed also throw san error if negated and converted to a Bayesian network. We should probably update that case as well, but I'm not sure what the correct value would be.

Reply:
I imagine it should be similar to conj with `Or.from_list(..)` and `self._unroll_disj(node)`? The latter does not exist but if I understand it correctly, it would be the same as `self.unroll_conj` with `== 'conj'` replaced with `=='disj'` ?